bin/bst-launch.ts

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+import * as program from "commander";
+import {Global} from "../lib/core/global";
+import {BSTAlexa} from "../lib/client/bst-alexa";
+
+program.version(Global.version());
+
+program
+    .usage("[options]")
+    .option("-u, --url <alexa-skill-url>", "The URL of the Alexa skill to speak to - defaults to current proxied skill")
+    .option("-a, --appId <application-id>", "The application ID for the skill")
+    .option("-U, --userId <user-id>", "Sets the user id to the specified value")
+    .option("-t, --accessToken <accessToken>", "Sets the access token for emulating a user with a linked account")
+    .description("Creates an launch request and sends it to your skill")
+    .action(function () {
+        // Just by casting program to options, we can get all the options which are set on it
+        const options: any = program;
+        let url = options.url;
+        const applicationID = options.appId;
+
+        if (options.url === undefined) {
```

Comment:
I would suggest to have simpler way to cast `undefined` and/or `null`:
```typescript
if (!options.url) {
}
```
Same in line 23.